Sources/PackageGraph/PackageGraphLoader.swift

```diff
@@ -273,10 +273,13 @@ private func createResolvedPackages(
             // If a target with similar name was encountered before, we emit a diagnostic.
             let targetName = targetBuilder.target.name
             if allTargetNames.contains(targetName) {
-                diagnostics.emit(ModuleError.duplicateModule(targetName), location: diagnosticLocation())
+                let targets = packageBuilders
+                .filter( { $0.targets
+                    .filter( { $0.target.name == targetName } ).count > 0 } )
```

Comment:
`.contains({ $0.target.name == targetName })` will be more performant (and IMHO clearer to read) than `.filter({ ... }).count > 0`

Reply:
Definitely, thanks!